scripts/autogen.py

```diff
@@ -34,10 +34,9 @@
 PROJECT_URL = {
     "keras": f"{KERAS_TEAM_GH}/keras/tree/v3.10.0/",
     "keras_tuner": f"{KERAS_TEAM_GH}/keras-tuner/tree/v1.4.7/",
-    "keras_hub": f"{KERAS_TEAM_GH}/keras-hub/tree/v0.20.0/",
+    "keras_hub": f"{KERAS_TEAM_GH}/keras-hub/tree/v0.21.1/",
     "tf_keras": f"{KERAS_TEAM_GH}/tf-keras/tree/v2.19.0/",
-    # TODO: Use the correct version when we cut a release.
-    "keras_rs": f"{KERAS_TEAM_GH}/keras-rs/tree/main/"
+    "keras_rs": f"{KERAS_TEAM_GH}/keras-rs/tree/v0.2.1/"
```

Comment:
1. you need to switch to v0.2.2
2. if you change this to not be `main`, you also need to change the `requirements.txt` file.

Reply:
Done